test-case/check-capture.sh

```diff
@@ -95,8 +95,7 @@ do
                     dlogi "using $file as capture output"
                 fi
 
-                dlogc "arecord -D$dev -r $rate -c $channel -f $fmt_elem -d $duration $file -v -q"
```

Comment:
The dlogc line is used to print the command we are running to logs, no reason to remove this.

Reply:
@aiChaoSONG the warpper `arecord_opts` did the dlogc already.

Reply:
@marc-hb sorry, my bad, ignore this comment.